models/project.ini

```diff
@@ -23,7 +23,7 @@ width = 1/4
 
 [fields.maturity]
 label = Maturity
-type = select
+type = checkboxes
```

Comment:
Yeah, that makes sense. 

I'm not sure if it's possible, but if we can use the model data to organise pages, that'd be good, cos otherwise the attic boolean does nothing. I know we can do that one deep, but not two, iirc. Will investigate